src-java/kilda-model/src/main/java/org/openkilda/model/Isl.java

```diff
@@ -116,9 +117,17 @@
     @Property(name = "under_maintenance")
     private boolean underMaintenance;
 
+    // TODO(surabujin): drop after migration
     @Property(name = "enable_bfd")
     private boolean enableBfd;
 
+    @Property("bfd_interval")
+    @Convert(graphPropertyType = Long.class)
+    private Duration bfdInterval;
+
+    @Property("bfd_multiplier")
+    private short bfdMultiplier;
+
     @Property(name = "bfd_session_status")
     @Convert(graphPropertyType = String.class)
     private BfdSessionStatus bfdSessionStatus;
```

Comment:
Please, double-check that property is worked as expected on the current master.

Reply:
Is there any specific scenario that causes this comment to appear here? I will review BFD "reportiong" tools one more time.